webapp/packages/plugin-data-viewer/src/TableViewer/TableHeader/TableHeaderMenu.tsx

```diff
@@ -14,7 +14,7 @@ import { useMenu } from '@cloudbeaver/core-view';
 
 import { DATA_CONTEXT_DV_DDM } from '../../DatabaseDataModel/DataContext/DATA_CONTEXT_DV_DDM.js';
 import { DATA_CONTEXT_DV_DDM_RESULT_INDEX } from '../../DatabaseDataModel/DataContext/DATA_CONTEXT_DV_DDM_RESULT_INDEX.js';
-import { DATA_CONTEXT_DATA_VIEWER_SIMPLE } from './DATA_CONTEXT_DATA_VIEWER_SIMPLE.js';
+import { DATA_CONTEXT_DV_SIMPLE } from '../../DatabaseDataModel/DataContext/DATA_CONTEXT_DV_SIMPLE.js';
```

Comment:
please move this context back to view part cuz model knows nothing about simple/not simple it's only for view